sympy/physics/mechanics/kane.py

```diff
@@ -132,62 +177,96 @@ def __init__(self, frame, q_ind, u_ind, kd_eqs=None, q_dependent=None,
         self._bodylist = None
 
         self._initialize_vectors(q_ind, q_dependent, u_ind, u_dependent,
-                u_auxiliary)
+                                 u_auxiliary)
+
         self._initialize_kindiffeq_matrices(kd_eqs)
+
         self._initialize_constraint_matrices(configuration_constraints,
-                velocity_constraints, acceleration_constraints)
+                                             velocity_constraints,
+                                             acceleration_constraints)
 
     def _initialize_vectors(self, q_ind, q_dep, u_ind, u_dep, u_aux):
-        """Initialize the coordinate and speed vectors."""
+        """Initialize the coordinate and speed vectors and sets the private
+        class attributes that contain them. The results are publicly accessible
+        through the attributes described in the class docstring.
 
-        none_handler = lambda x: Matrix(x) if x else Matrix()
+        """
 
         # Initialize generalized coordinates
-        q_dep = none_handler(q_dep)
-        if not iterable(q_ind):
-            raise TypeError('Generalized coordinates must be an iterable.')
+        q_dep = _none_handler(q_dep)
         if not iterable(q_dep):
             raise TypeError('Dependent coordinates must be an iterable.')
+        if not iterable(q_ind):
+            raise TypeError('Generalized coordinates must be an iterable.')
```

Comment:
Since q_dep and q_ind are both generalized coordinates should the strings read "Dependent generalized coordinates" and "Independent generalized coordinates"?